x/examples/outline-connectivity/main.go

```diff
@@ -159,6 +210,28 @@ func main() {
 			if err != nil {
 				log.Fatalf("Failed to output JSON: %v", err)
 			}
+			// Send error report to collector if specified
+			if *reportToFlag != "" {
+				var samplingRate float64
+				if !success {
+					samplingRate = *reportFailureFlag
+				} else {
+					samplingRate = *reportSuccessFlag
+				}
+				// Generate a random number between 0 and 1
+				random := rand.Float64()
+				if random < samplingRate {
+					// Run your function here
+					err = sendReport(record, *reportToFlag)
+					if err != nil {
+						log.Fatalf("HTTP request failed: %v", err)
```

Comment:
This would be clearer:
```suggestion
						log.Fatalf("Report failed: %v", err)
```